frontend/src/components/authentication/CreateAccountPage/CreateAccountPage.tsx

```diff
@@ -77,10 +85,12 @@ const CreateAccountPage = (): JSX.Element => {
         }).catch(
             error => {
                 const code = error.code;
+                debugger;
                 console.log(error);
+                setPasswordError(error.message);
                 switch (code) {
                     case 'UsernameExistsException':
```

Comment:
This is good that you're setting the message to the `error.message` just please clean up debugger lines and console logs
```suggestion
                setPasswordError(error.message);
                switch (code) {
                    case 'UsernameExistsException':
```

For more info on AWS signUp errors, you can check out this link if you haven't already [https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_SignUp.html#errors:~:text=Type%3A%20String-,Errors,-For%20information%20about](url)

Reply:
The switch statement might be pointless, since all errors should stop it anyway. Check and make sure I'm if I'm correct or not tho